services/distance.md

```diff
@@ -0,0 +1,30 @@
+# Distance
+
+A sensor that determines the distance of an object without any physical contact involved.
+
+        identifier:  0x141a6b8a
+        extends: _sensor
+
+## Registers
+
+    ro distance: u6.10 m { typical_min=0.02, typical_max=4 } @ reading
```

Comment:
maybe we should standardize on u16.16 when there are no good to reasons to do otherwise? there is almost no transfer cost difference (and often literally none as packets in frames are padded to full word) and it gives us more range and precision.

Reply:
fixed